### PR TITLE
Fixing DB version upgrade

### PR DESCRIPTION
**Pull request by joshcanhelp**
- Upgrading Lock 11.0.0 => 11.1
- Upgrading Auth.js 9.0.0 => 9.1
- Checking for an existing client grant for the management API before failing
- Admin notice if client grant is successful to info and catch edge cases

Waiting for #366 to be completed before submitting

### CODE REVIEW

File: lib/WP_Auth0_DBManager.php [hunk 1]
```diff
@@ -259,6 +274,24 @@ public function notice_failed_client_grant() {
...
 
+	public function notice_successful_client_grant() {
+
+		if ( ! get_option( 'wp_auth0_client_grant_success' ) ) {
+			return;
+		}
+		?>
+		<div class="notice notice-success">
+			<p><?php
+				_e( 'As a part of the 3.4.1 upgrade, a client grant was created for the Auth0 Management API.', 'wp-auth0' );
```
Comment (cocojoe) [inline lib/WP_Auth0_DBManager.php]: I think this should be 3.5.0 now, there are a lot of fixes in this release so it's significant to warrant, same for all 3.4.1 references.

File: lib/WP_Auth0_Api_Client.php [hunk 2]
```diff
@@ -550,7 +550,7 @@ public static function create_client_grant( $app_token, $client_id ) {
 		$data = array(
 			'client_id' => $client_id,
 			'audience' => self::get_connect_info( 'audience' ),
-			'scope' => self::ConsentRequiredScopes()
+			'scope' => self::get_required_scopes()
```
Comment (cocojoe) [inline lib/WP_Auth0_Api_Client.php]: this add in the last PR?

Comment (joshcanhelp) [reply lib/WP_Auth0_Api_Client.php]: @cocojoe - I'm not sure what you're asking here.

Comment (cocojoe) [reply lib/WP_Auth0_Api_Client.php]: Point me to this method please

Comment (joshcanhelp) [reply lib/WP_Auth0_Api_Client.php]: @cocojoe - Couldn't find it? 😆

https://github.com/auth0/wp-auth0/blob/master/lib/WP_Auth0_Api_Client.php#L221

**Pull request merged**